ROADMAP.md

```diff
@@ -4,4 +4,138 @@ Roadmap
 _Note: Previous roadmaps can be found with the roadmap reviews for that period.
 [Link to Roadmap 2022](roadmap-reviews/2022/ROADMAP.md)_
 
-2023 roadmap coming soon.
\ No newline at end of file
+This document spans the Roadmap for the in-toto project for the 2023 calendar
+year. As before, the theme of this year's efforts are focused on extending
+in-toto through new and existing ITEs, and adding support for them in the
+implementations we maintain.
+
+## in-toto v1.0
+
+The last tagged release of the in-toto specification was v0.9. We are gearing
+up to tag v1.0 of the specification. This process entails addressing some long
+standing discussions and issues, and several rounds of reviews to catch any
+inconsistencies and errors that may require breaking changes to fix later.
+Apart from these processes, there aren't any major blockers to the
+specification reaching the 1.0 milestone -- it has been stable for a significant
+amount of time and much of in-toto's development happens through in-toto
+Enhancements (ITEs). We expect the v1.0 release to happen in the first review
+period of this roadmap.
+
+## ITEs
+
+Several important ITEs were introduced in the last few years. ITEs 2 and 3
+discussed how to use TUF as a root of trust for in-toto metadata. ITE-4
+introduced the notion of non-file artifacts in in-toto, a semantic that is very
+useful in capturing attestations for real world software supply chains. ITE-5
+disassociated the signature wrapper from the in-toto specification and
+recommended a switch to DSSE. ITE-6 perhaps had the most impact of all the
+ITEs, introducing the in-toto Attestation framework. This was paired with ITE-9
+that described a process for the in-toto community to introduce new attestation
+types, and established the in-toto attestation maintainers, a group of industry
+stakeholders who review and provide feedback about new attestation types. ITE-7
+added support for creating and verifying signatures using X.509 certificates.
```

Comment:
What happened to ITE-8?

Also, consider adding links to the ITEs?

Reply:
ITE-8 is mentioned in the section about updated layouts as it needs a bit of a revival once we have policy capabilities for attestations. Added links to other ITEs.